gammapy/data/event_list.py

```diff
@@ -653,6 +653,47 @@ def plot_time_map(self, ax=None):
 
         return ax
 
+    def plot_theta2_distribution(self, ax=None, number_bins=50):
+        """Plot the theta2 distribution of the events.
+        A pointing direction can be given in the same units as the event list as (ra,dec).
+        If None is given, the average of incoming directions of the events is taken.
```

Comment:
Docstring is incorrect. You currently don't take a pointing direction argument.

Reply:
Thank you so much for taking the time to comment in details and for your suggestions to improve this function!